lib/perl/Genome/Model/Tools/CopyNumber/Cnmops.pm

```diff
@@ -199,12 +204,22 @@ sub annotate_cnvs {
   }
 }
 
+sub _bedtools_path {
```

Comment:
This logic already exists in [Genome::Model::Tools::Bwa::RunMem](https://github.com/genome/genome/blob/fcb7058fe1aa98d03997d5600cb694d94ea7e95f/lib/perl/Genome/Model/Tools/Bwa/RunMem.pm#L205) which means this is duplication and instead of doing that you should centralize it.  So it seems like this should be implemented in `Genome::Model::Tools::BedTools`.